xblock/core.py

```diff
@@ -5,11 +5,12 @@
 and used by all runtimes.
 
 """
+from collections import defaultdict
```

Comment:
They aren't new imports, just reordering of the existing ones.  I'll revert if it causes merge problems.